### ISSUE TITLE
Error in the console

### ISSUE THREAD

**Issue by madvik** [WordPress/gutenberg#3950]
WordPress version : 4.9.1
Gutenberg version : 1.9
Plugins : just Gutenberg
## Issue Overview
Here: 
```
mce-view.min.js?ver=4.9.1:1 Uncaught TypeError: Cannot read property 'clientWidth' of undefined
    at n.initialize (https://codetheworld.info/wp-includes/js/mce-view.min.js?ver=4.9.1:1:8267)
    at n.b.mce.View (https://codetheworld.info/wp-includes/js/mce-view.min.js?ver=4.9.1:1:1732)
    at new n (https://codetheworld.info/wp-admin/load-scripts.php?c=0&load%5B%5D=jquery-core,jquery-migrate,utils,underscore,backbone,wp-api-request,wp-util,wp-backbone,media-models,moxiejs,plupload,wp-pluploa&load%5B%5D=d,jquery-ui-core,jquery-ui-widget,jquery-ui-mouse,jquery-ui-sortable&ver=4.9.1:17:22931)
    at Object.createInstance (https://codetheworld.info/wp-includes/js/mce-view.min.js?ver=4.9.1:1:1182)
    at https://codetheworld.info/wp-includes/js/mce-view.min.js?ver=4.9.1:1:551
    at Function.m.each.m.forEach (https://codetheworld.info/wp-admin/load-scripts.php?c=0&load%5B%5D=jquery-core,jquery-migrate,utils,underscore,backbone,wp-api-request,wp-util,wp-backbone,media-models,moxiejs,plupload,wp-pluploa&load%5B%5D=d,jquery-ui-core,jquery-ui-widget,jquery-ui-mouse,jquery-ui-sortable&ver=4.9.1:15:2350)
    at https://codetheworld.info/wp-includes/js/mce-view.min.js?ver=4.9.1:1:368
    at Function.m.each.m.forEach (https://codetheworld.info/wp-admin/load-scripts.php?c=0&load%5B%5D=jquery-core,jquery-migrate,utils,underscore,backbone,wp-api-request,wp-util,wp-backbone,media-models,moxiejs,plupload,wp-pluploa&load%5B%5D=d,jquery-ui-core,jquery-ui-widget,jquery-ui-mouse,jquery-ui-sortable&ver=4.9.1:15:2410)
    at Object.setMarkers (https://codetheworld.info/wp-includes/js/mce-view.min.js?ver=4.9.1:1:328)
    at q.<anonymous> (https://codetheworld.info/wp-includes/js/tinymce/plugins/wpview/plugin.min.js?wp-mce-4607-20171116:1:1154)
    at b.e [as fire] (https://codetheworld.info/wp-includes/js/tinymce/tinymce.min.js?ver=4607-20171116:27:15281)
    at q.fire (https://codetheworld.info/wp-includes/js/tinymce/tinymce.min.js?ver=4607-20171116:27:17182)
    at q.setContent (https://codetheworld.info/wp-includes/js/tinymce/tinymce.min.js?ver=4607-20171116:34:9565)
    at q.<anonymous> (https://codetheworld.info/wp-content/plugins/gutenberg/blocks/build/index.js?ver=1513086555:12:245275)
    at b.e [as fire] (https://codetheworld.info/wp-includes/js/tinymce/tinymce.min.js?ver=4607-20171116:27:15281)
    at q.fire (https://codetheworld.info/wp-includes/js/tinymce/tinymce.min.js?ver=4607-20171116:27:17182)
    at q.load (https://codetheworld.info/wp-includes/js/tinymce/tinymce.min.js?ver=4607-20171116:34:8926)
    at Object.D [as initContentBody] (https://codetheworld.info/wp-includes/js/tinymce/tinymce.min.js?ver=4607-20171116:33:24129)
    at Object.r [as init] (https://codetheworld.info/wp-includes/js/tinymce/tinymce.min.js?ver=4607-20171116:33:28187)
    at q.<anonymous> (https://codetheworld.info/wp-includes/js/tinymce/tinymce.min.js?ver=4607-20171116:33:29445)
    at Array.<anonymous> (https://codetheworld.info/wp-includes/js/tinymce/tinymce.min.js?ver=4607-20171116:4:23892)
    at b (https://codetheworld.info/wp-includes/js/tinymce/tinymce.min.js?ver=4607-20171116:2:26780)
    at r (https://codetheworld.info/wp-includes/js/tinymce/tinymce.min.js?ver=4607-20171116:4:23839)
    at l.(anonymous function).l.(anonymous function).l.(anonymous function).l.(anonymous function) (https://codetheworld.info/wp-includes/js/tinymce/tinymce.min.js?ver=4607-20171116:4:23773)
    at HTMLScriptElement.f (https://codetheworld.info/wp-includes/js/tinymce/tinymce.min.js?ver=4607-20171116:4:22631)
```

## Steps to Reproduce (for bugs)
1. Go in an article in edition
2. Open the console

**Comment by westonruter** [comment]
I think this is a core issue: https://core.trac.wordpress.org/ticket/42416

**Comment by bradyvercher** [comment]
I think this is a duplicate of #3568. In any case, this bug is preventing me from opening existing posts with oEmbeds in the Gutenberg editor.

**Comment by westonruter** [comment]
I've committed the [fix](https://core.trac.wordpress.org/changeset/42390) in WordPress core to close https://core.trac.wordpress.org/ticket/42416. So you can update to the next nightly for trunk or the 4.9 branch to resolve the issue. The fix will be included in 4.9.2 next month.
